static/src/javascripts/projects/commercial/modules/messenger/background.js

```diff
@@ -0,0 +1,100 @@
+// @flow
+import { addEventListener } from 'lib/events';
+import fastdom from 'lib/fastdom-promise';
+
+import { register } from 'commercial/modules/messenger';
+
+type AdSpec = {
+    scrollType: string,
+    backgroundColor: string,
+    backgroundImage: string,
+    backgroundRepeat: string,
+    backgroundPosition: string,
+};
+
+const setBackground = (specs: AdSpec, adSlot: Node): Promise<any> | null => {
```

Comment:
We could probably make this a [`Maybe`](https://flow.org/en/docs/types/maybe/) type too: `?Promise<any>`